decidim-admin/app/views/decidim/admin/users_statistics/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="content">
+  <div class="row">
+    <%= render partial: "decidim/admin/users_statistics/users_count", locals: { result: result } %>
```

Comment:
Is this file reaelly used? I can't find where!

Reply:
I think @mrcasals is right @Leusev , who is rendering this `index` file ?

Reply:
Yes, you're right @mrcasals, I've realized I added that template by mistake.
It was an initially try to do create an index template that finally I didn't used
Thanks!